test/setup.js

```diff
@@ -221,14 +221,14 @@ function cleanDatabase(assert, cb) {
   var db = new DB(testAppFormsDb, new Server(testsConfig.dbAddress, testsConfig.dbPort), {"w": 1, j: false});
 
   db.open(function (err, db) {
-    assert.ok(!err);
+    assert.ok(!err, err);
     assert.ok(db);
 
     db.authenticate("admin", "admin", {"authSource": "admin"}, function (err, ok) {
-      assert.ok(!err);
+      assert.ok(!err,err);
```

Comment:
Put a space after the comma, here and on line 231. I'd have hoped that eslint would pick up on that :-/ Maybe it's not linting the test files